bot/commands/imagesearch.js

```diff
@@ -16,19 +16,31 @@ exports.init = function () {
 
             otterbot.log('Searching for an image: ' + message);
 
-            request(url, function (err, res, body) {
-                if (!err && res.statusCode == 200) {
-                    var images = JSON.parse(body)["responseData"]["results"];
+            var fetchImage = function(images) {
```

Comment:
@jkupferman Wondering if this would be more helpful abstracted out into a helper as we may have use for this elsewhere. Basically a function that would take a url, could return a `Promise` (don't have any of these elsewhere but I think it's a good fit for it), which would reject on error or non-200, and resolve with the url on success. Could then do the `chatSingle` in the resolve.

Let me know what you think, I could probably work through that this week if you dig it.